poly/main.go

```diff
@@ -95,7 +95,7 @@ func application() *cli.App {
 					&cli.StringFlag{
 						Name:  "i",
 						Value: "",
-						Usage: "Specify file input type. Options are Gff, gbk/gb, and json. Defaults to none.",
+						Usage: "Specify file input type. Options are gff, gbk/gb, fasta, and json. Defaults to none. For use with pipes.",
```

Comment:
Another place that maybe good to throw an error is if no valid `-i` value is supplied when piping.

(Also, is `Defaults to none` a valid case?)

Reply:
Definitely not a valid case while piping. You're right that it should be throwing an error in this case @jkhales.